tests/chainerx_tests/array_utils.py

```diff
@@ -112,6 +112,13 @@ def create_dummy_ndarray(
     return a
 
 
+def dropout(a, prob=0.5):
```

Comment:
This function exists only for tests for `chainrex.all` and `chainerx.any`. How about moving this function to `test_logic.py`?

Reply:
Yeah it is only used by those two. But I thought it may come in handy for other functions as well maybe?.

Will move it to `test_logic.py` for now.